tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/TestingFrameworkImpactedTests.cs

```diff
@@ -243,6 +242,26 @@ private void InitGit()
                 output = RunGitCommand("branch --show-current");
             }
 
+            if (output.ExitCode == 0)
+            {
+                // Retrieve branch name
+                branch = output.Output.Trim();
+            }
+
+            if (output.ExitCode == 0)
+            {
+                // Retrieve last commit
+                output = RunGitCommand("rev-parse --verify HEAD");
+                if (output.ExitCode == 0)
```

Comment:
If this fails, you now no-longer run the `RunGitCommand("rev-parse --show-toplevel");` block below. Are you sure that's what you want to do?